test/ctor-eval/gc-2.wast.out

```diff
@@ -1,5 +1,5 @@
 (module
- (type $struct (struct (field i32)))
+ (type $struct (sub (struct (field i32))))
```

Comment:
Why did this change?

Reply:
This is a non-final type. Now that we interpret the shorthand `(type $... (struct ...))` as a final type, we need to add `(sub ...)` to the printed output to correctly communicate that this is a non-final type.

Reply:
I'm still confused then, sorry. Why is this non-final? Final is the default, and I don't see "open" here, so isn't it final?

Reply:
Right now Binaryen's text format does not include any of the changes from https://github.com/WebAssembly/gc/pull/413, so `(type (struct ...))` is final, `(type (sub (struct ...)))` is open, and `(type (sub final (struct ...)))` is final. Yes, this is confusing, so we're going to fix it in that PR.